tests/test_integration.py

```diff
@@ -1338,3 +1339,30 @@ def test_issues_539_abi3_resolution():
     res.assert_success()
 
     subprocess.check_call([cryptography_pex, '-c', 'import cryptography'])
+
+
+@pytest.mark.skip("Acceptance test for landing https://github.com/pantsbuild/pex/issues/716.")
+def test_reproducible_build():
+
+  def assert_reproducible(args):
+    with temporary_dir() as td:
+      pex1 = os.path.join(td, '1.pex')
+      pex2 = os.path.join(td, '2.pex')
+      run_pex_command(args + ['-o', pex1])
+      run_pex_command(args + ['-o', pex2])
+      assert filecmp.cmp(pex1, pex2, shallow=False)
+
+  assert_reproducible([])
+
+  # With pre-built wheels, including
+  # 1) a universal wheel - six
+  # 2) an abi wheel - cryptography
+  assert_reproducible(['six==1.12.0', 'cryptography==2.6.1'])
+
+  # With an sdist
+  assert_reproducible(['pycparser==2.19', '--no-build'])
+
+  assert_reproducible(['-m', 'pydoc'])
+  assert_reproducible(['black==19.3b0', '-c', 'black'])
```

Comment:
I'd love to use a command that does not involve PyPi, since we already test that above. However, I can't think of a command that involves `-c` and does not first download a dependency. Let me know if you know of anything.

Reply:
Sure, you can build a PEX with just sources and no deps by either pointing at a local setup.py - you spell the requirement as the path of the dir containing setup.py - or use ... `--sources-directory` & `--resources-directory`.